docs/howto/README.md

```diff
@@ -4,6 +4,12 @@ Here you’ll find short answers to "How do I...?" types of questions. These how
 topics in depth – you’ll find that material in the [topic guides](../topics/README.md) section.
 However, these guides will help you quickly accomplish common tasks.
 
+Generic approach for any Kubernetes cluster (using inlets):
```

Comment:
Can we place this after the provider-specific HOWTOs? In general, it's best for the reader to find examples specific to the cloud provider/product, then try the generic example if they're feeling brave.

Reply:
From my perspective the AKS VM creation is really rather complicated and verbose when this approach could save users time getting started. That's the only reason I thought the generic approach may be better placed first. 

Since we only use kubectl port-forward and kubectl apply it will work with any cluster including AKS. 

From an onboarding perspective (as a newcomer to the project), I'd much rather use a genetic approach than having to set up a VM etc. To some extent the other guides become redundant. 

That said, it's your repo and project, so whatever you feel is easier for users to get started with.

Reply:
I see where you're coming from. That being said, instructions specific to a particular cloud provider or project might be able to recommend different products or tools that may be better suited to their environment's needs. For example @thomastaylor312's guide demonstrates how to re-use components that were already part of the cluster like the vnet and the resource group, resulting in a cleaner setup.